src/NexusMods.App.UI/Controls/GameWidget/GameWidget.axaml.cs

```diff
@@ -19,58 +21,64 @@ public GameWidget()
             {
                 this.WhenAnyValue(view => view.ViewModel!.State)
                     .Subscribe(state =>
-                    {
-                        DetectedGameStackPanel.IsVisible = state == GameWidgetState.DetectedGame;
-                        AddingGameStackPanel.IsVisible = state == GameWidgetState.AddingGame;
-                        ManagedGameGrid.IsVisible = state == GameWidgetState.ManagedGame;
-                        RemovingGameStackPanel.IsVisible = state == GameWidgetState.RemovingGame;
-                        GameWidgetBorder.Classes.ToggleIf("Disabled", state is GameWidgetState.AddingGame or GameWidgetState.RemovingGame);
-                    })
+                        {
+                            DetectedGameStackPanel.IsVisible = state == GameWidgetState.DetectedGame;
+                            AddingGameStackPanel.IsVisible = state == GameWidgetState.AddingGame;
+                            ManagedGameGrid.IsVisible = state == GameWidgetState.ManagedGame;
+                            RemovingGameStackPanel.IsVisible = state == GameWidgetState.RemovingGame;
+                            GameWidgetBorder.Classes.ToggleIf("Disabled", state is GameWidgetState.AddingGame or GameWidgetState.RemovingGame);
+                        }
+                    )
                     .DisposeWith(d);
 
+
                 this.OneWayBind(ViewModel, vm => vm.Image, v => v.GameImage.Source)
                     .DisposeWith(d);
 
-                this.WhenAnyValue(view => view.ViewModel!.Image)
-                    .WhereNotNull()
-                    .OffUi()
-                    .Select(BlurAvaloniaImage)
-                    .OnUI()
-                    .BindToView(this, view => view.BlurryImage.Source)
+                this.OneWayBind(ViewModel, vm => vm.Name, v => v.NameTextBlock.Text)
                     .DisposeWith(d);
 
-                this.OneWayBind(ViewModel, vm => vm.Name, v => v.NameTextBlock.Text)
+                this.OneWayBind(ViewModel, vm => vm.Version, v => v.VersionTextBlock.Text)
+                    .DisposeWith(d);
+
+                this.OneWayBind(ViewModel, vm => vm.GameStoreIcon, view => view.GameStoreIcon.Value,
+                        image => MapGameStoreToIconClass(ViewModel!.Installation.Store)
+                    )
                     .DisposeWith(d);
 
```

Comment:
This is odd. Since `GameStoreIcon` is `IconValue` you should just do this:

```csharp
                this.OneWayBind(ViewModel, vm => vm.GameStoreIcon, view => view.GameStoreIcon)
                    .DisposeWith(d);
```

In the ViewModel, you'd wanna do this:

```csharp
            this.WhenAnyValue(vm => vm.Installation)
                .Select(installation => MapGameStoreToIcon(installation.Store))
                .BindToVM(this, vm => vm.GameStoreIcon)
                .DisposeWith(disposables);
```